mathics/builtin/system.py

```diff
@@ -455,3 +455,121 @@ def evaluate(self, evaluation) -> Real:
         # Make this be whatever the latest Mathematica release is,
         # assuming we are trying to be compatible with this.
         return Real(self.value)
+
+
+class SystemMemory(Predefined):
+    """
+    <dl>
+      <dt>'$SystemMemory'
+      <dd>Returns the total amount of physical memory.
+    </dl>
+
+    >> $SystemMemory
+     = ...
+    """
+
+    name = "$SystemMemory"
+    
+    def evaluate(self, evaluation) -> Integer:
+        try:
+            import psutil
+            totalmem = psutil.virtual_memory().total
+            return Integer(total)
```

Comment:
@mmatera there is a typo here and below. I believe `total` should be `totalmem` which is set on line 476. 

I will be addressing this in another PR.